Sources/_FoundationCShims/include/_CStdlib.h

```diff
@@ -122,9 +122,15 @@
 #include <complex.h>
 #endif
 
+#if !defined(_WIN32)
+// On windows, atomic_signal_fence from clang headers is incompatible
+// with atomic_signal_fence definition from MSVC when C++ interop is enabled,
+// so drop the stdatomic include.
+// Works around: https://github.com/swiftlang/swift/issues/75720
 #if __has_include(<stdatomic.h>)
 #include <stdatomic.h>
```

Comment:
Presumably this is in here because something in swift-foundation requires it - do we know what that is / what will be broken by not having this include? Just want to confirm which scenario we have:

1. This import isn't used at all - in which case we should just remove it instead of conditionalizing it
2. This import isn't used on Windows but is used on other platforms - in which case this is fine, but it'd be good to know what needs it
3. This import is used on all platforms - in which case CI should hopefully show us where Windows needs it

Reply:
Good questions! Let me investigate.

Reply:
It looks like we can drop `stdatomic` include alltogether, and nothing in swift-foundation uses, and swift-corelibs-foundation includes it too in CFInternal.h . Let me just do some more testing and I'll update this PR removing it alltogether.

Reply:
Removed the include!